robottelo/hosts.py

```diff
@@ -868,6 +868,10 @@ def is_upstream(self):
         """
         return self.execute('rpm -q satellite &>/dev/null').status != 0
 
+    @cached_property
```

Comment:
This is duplicating the cached property on the Satellite class.

Please remove the function definition from Satellite if it should be here on Capsule - since Satellite inherits from Capsule.